packages/component/README.md

```diff
@@ -2,19 +2,3 @@
 
 _NOTE: This is specifically tailored for Firebase JS SDK usage, if you are not a
 member of the Firebase team, please avoid using this package_
-
-## Installation
```

Comment:
I think it's okay to keep this section for internal packages in order to provide instruction for Googlers. We have clearly stated that this package is intended to be only used by the Firebase team, so I don't think there is any confusion.

Reply:
I restored Usage but kept Installation removed because I don't think any internal user would (or should) npm install it?